epiquote/settings/conf.sample.py

```diff
@@ -18,3 +19,21 @@
         'NAME': 'epiquote.db',
     }
 }
+
+BASE_DIR = os.getcwd()
+
+TEMPLATES = [
```

Comment:
This should go in common, not in conf.sample.py, as this isn't something we want users to configure.